### ISSUE TITLE
getVideoComments gives error "commentThreads not found"

### ISSUE THREAD

**Issue by phazonoverload** [jasonhaxstuff/popyt#367]
### What behaviour is expected

Running the following code snippet should return a list of comments for the given video:

```js
const comments = await youtube.getVideoComments(Video, 0)
```

### What behaviour is observed

I get the error `commentThreads not found`

### Additional context

Full snippet:

```js
const channel = await youtube.getChannel(process.env.YOUTUBE_CHANNEL_ID)
const uploadsPlaylistId = channel.data.contentDetails.relatedPlaylists.uploads
const playlist = await youtube.getPlaylistItems(uploadsPlaylistId, 0)

for(let video of playlist) {
      const comments = await youtube.getVideoComments(video, 0)
      console.log(comments)
}
```

**Comment by jasonhaxstuff** [comment]
Also, I forgot to mention, since this method works when directly using an ID (according to the test suite), you should be able to use `youtube.getVideoComments(video.id, 0)` instead and have it work. If this doesn't work, maybe this has something to do with one of the videos having comments disabled.

**Comment by phazonoverload** [comment]
Exactly what I ended up doing! Thanks again - I previously wrote all this functionality by-hand and this is so much nicer. Feel free to close the issue if you want.

**Comment by jasonhaxstuff** [comment]
Fixed in 4ca35d8
